commons-oauth/src/main/java/eu/europeana/api/commons/oauth2/model/impl/EuropeanaApiCredentials.java

```diff
@@ -25,16 +26,22 @@ public EuropeanaApiCredentials(String userName, String clientId) {
     this.clientId = clientId;
 
   }
-  public EuropeanaApiCredentials(String userName, String clientId,String apikey) {
+  public EuropeanaApiCredentials(String userName, String clientId,String apikey, String affiliation) {
     this.userName = userName;
     this.clientId = clientId;
     this.apiKey = apikey;
+    this.affiliation = affiliation;
```

Comment:
should generate a setter method as well, as this is not a mandatory field

Reply:
A setter method is not used anywhere at the moment so I left it without it. If needed please create.

Reply:
if the related apis need to overwrite the affiliation, they must have the possibility to do so ... The mandatory fields can stay without setter, but the optional ones should always have a setter method.